solarforecastarbiter/io/reference_observations/arm_reference_sites.json

```diff
@@ -0,0 +1,128 @@
+            "timezone": "America/Chicago",
+            "extra_parameters": "{\"observation_interval_length\": 1.0, \"network_api_abbreviation\": \"sgp\", \"attribution\": \"https://www.arm.gov/capabilities/vaps/qcrad\", \"network\": \"DOE ARM\", \"arm_site_id\": \"sgp\", \"network_api_id\": \"E12\", \"datastreams\": {\"qcrad\": {\"sgpqcrad1longE12.c2\": \"1996-01-19/2019-08-26\" ,\"sgpqcrad1longE12.c1\": \"2019-08-26/now\"}}}"
+        },
+        {
+            "name": "E33: Newkirk, Oklahoma",
+            "latitude": 36.925498962402344,
+            "longitude": -97.08170318603516,
+            "elevation": 356.0,
+            "timezone": "America/Chicago",
+            "extra_parameters": "{\"observation_interval_length\": 1.0, \"network_api_abbreviation\": \"sgp\", \"attribution\": \"https://www.arm.gov/capabilities/vaps/qcrad\", \"network\": \"DOE ARM\", \"arm_site_id\": \"sgp\", \"network_api_id\": \"E33\", \"datastreams\": {\"met\": \"sgpmetE33.b1\", \"qcrad\": {\"sgpqcrad1longE33.c2\": \"2011-08-17/2017-07-14\", \"sgpqcrad1longE33.c1\": \"2017-07-14/now\"}}}"
+        },
+        {
+            "name": "E37: Waukomis, Oklahoma",
+            "latitude": 36.31060028076172,
+            "longitude": -97.9280014038086,
+            "elevation": 387.0,
+            "timezone": "America/Chicago",
+            "extra_parameters": "{\"observation_interval_length\": 1.0, \"network_api_abbreviation\": \"sgp\", \"attribution\": \"https://www.arm.gov/capabilities/vaps/qcrad\", \"network\": \"DOE ARM\", \"arm_site_id\": \"sgp\", \"network_api_id\": \"E37\", \"datastreams\": {\"met\": \"sgpmetE37.b1\", \"qcrad\": {\"sgpqcrad1longE37.c2\": \"2011-09-28/2019-06-21\", \"sgpqcrad1longE37.c1\": \"2019-06-21/now\"}}}"
+        },
+        {
+            "name": "E9: Ashton, Kansas",
+            "latitude": 37.132999420166016,
+            "longitude": -97.26599884033205,
+            "elevation": 372.0,
+            "timezone": "America/Chicago",
+            "extra_parameters": "{\"observation_interval_length\": 1.0, \"network_api_abbreviation\": \"sgp\", \"attribution\": \"https://www.arm.gov/capabilities/vaps/qcrad\", \"network\": \"DOE ARM\", \"arm_site_id\": \"sgp\", \"network_api_id\": \"E9\", \"datastreams\": {\"met\": \"sgpmetE9.b1\", \"qcrad\": { \"sgpqcrad1longE9.c2\" :\"1994-01-12/2018-05-31\",  \"sgpqcrad1longE9.c1\": \"2018-05-31/now\"}}}"
+        },
+        {
+            "name": "North Slope of Alaska (NSA), Barrow, Alaska",
+            "latitude": 71.322998046875,
+            "longitude": -156.60899353027344,
+            "elevation": 5.0,
+            "timezone": "America/Anchorage",
+            "extra_parameters": "{\"observation_interval_length\": 1.0, \"network_api_abbreviation\": \"nsa\", \"attribution\": \"https://www.arm.gov/capabilities/vaps/qcrad\", \"network\": \"DOE ARM\", \"arm_site_id\": \"nsa\", \"network_api_id\": \"C1\", \"datastreams\": {\"met\": \"nsametC1.b1\", \"qcrad\": \"nsaqcrad1longC1.c1\"}}"
```

Comment:
does qcrad have to be a dict?

Reply:
It does not. When it's just a single string it skips the date range handling and attempts to request data for the whole period. For this NSA C1 site in particular, there is just one continuous qcrad datastream. This does make me think about logging periods of data that aren't available.